packages/react/src/components/tabs/tabs.tsx

```diff
@@ -142,14 +147,33 @@ export const Tabs: VoidFunctionComponent<Props> = ({
     forceRenderTabPanels,
     tabs,
     defaultSelectedId,
-    onAddTab,
+    addButtonProps: providedAddButtonProps,
     onRemove,
 }) => {
     const { t } = useTranslation('tabs');
     const tabsListRef = createRef<HTMLDivElement>();
     const scrollLeftButtonRef = createRef<HTMLButtonElement>();
     const scrollRightButtonRef = createRef<HTMLButtonElement>();
 
+    const addButtonProps = {
+        label: t('addTab'),
+        disabled: false,
+        loading: false,
+        tooltipContent: null,
+        ...providedAddButtonProps,
+    };
+    const addButtonComponent = (
+        <AddButton
+            type="button"
+            buttonType="tertiary"
+            leftIconName="plusSign"
+            label={addButtonProps.label}
+            disabled={addButtonProps.disabled}
+            loading={addButtonProps.loading}
+            onClick={addButtonProps.onClick}
+        />
+    );
```

Comment:
Tu pourrais peut-être mettre un conditionel ici pour pas le créér (ou dépendre de `addButton?.onClick &&`)
```suggestion
    const addButtonComponent = providedAddButtonProps && (
        <AddButton
            type="button"
            buttonType="tertiary"
            leftIconName="plusSign"
            label={addButtonProps.label}
            disabled={addButtonProps.disabled}
            loading={addButtonProps.loading}
            onClick={addButtonProps.onClick}
        />
    );
```